src/components/business-lookup/BusinessLookup.vue

```diff
@@ -18,8 +17,9 @@
         :items="searchResults"
         :loading="state == States.SEARCHING"
         :hide-no-data="state != States.NO_RESULTS"
+        @update:search-input="onSearchInput($event)"
```

Comment:
Linting changed the original code (`search-input:sync`) to what you see on old line 6, which doesn't work.

Instead of going back to the original code, I use this event handler.